integration-tests/src/test/java/io/testerra/test/pretest_status/DataProviderTest.java

```diff
@@ -30,20 +29,30 @@
 public class DataProviderTest extends TesterraTest {
 
     @Test(dataProviderClass = MyDataProvider.class, dataProvider = "throwException")
-    @Fails
-    public void test_interceptCrashedDataProvider() {
+    public void testT01_interceptCrashedDataProvider() {
         Assert.assertTrue(true);
     }
 
     @DataProvider()
     public Object[][] crashingDataProvider() {
```

Comment:
Given that we already use `eu.tsystems.mms.tic.testframework.test.common.MyDataProvider#throwException`, should this method not also be there? Or, alternatively, should `MyDataProvider` not be inlined?

Reply:
Why should this method added to `MyDataProvider`?

Reply:
Well, I figured: Either the test _uses_ an external data provider or it _does not use_ one. If mixing _is the point_, that's not visible in the method names and should probably be reflected either there or in JavaDoc.

Reply:
I added some comments.